obelisk_ws/src/robots/achilles_model/mujoco/achilles.xml

```diff
@@ -0,0 +1,303 @@
+<!-- <?xml version="1.0" encoding="utf-8"?> -->
+<mujoco model="achilles">
+
+  <!-- Set the angles to be in radians -->
+  <compiler angle="radian" autolimits="true" eulerseq="zyx" meshdir="./meshes"/>
+
+  <!-- Fix simulation options -->
+  <option timestep="0.001" iterations="50" solver="Newton" gravity="0 0 -9.81"  integrator = "RK4"/>
```

Comment:
Are you sure about all these settings?

- 0.001 is a very aggressive physics step, the default is 0.002
- I think `iterations` might be 50 by default, but you should double check that
- Newton is the default solver
- this gravity is default
- the recommended integrator is `implicitfast`, but you should read [the docs](https://mujoco.readthedocs.io/en/stable/computation/index.html#numerical-integration) to see their comments on when you should choose a different one